socorro/external/es/crashstorage.py

```diff
@@ -498,14 +498,10 @@ def save_processed_crash(self, raw_crash, processed_crash):
         es_doctype = self.get_doctype()
         all_valid_keys = self.get_keys(index_name, es_doctype)
 
-        src = {
-            "raw_crash": copy.deepcopy(raw_crash),
```

Comment:
One less deepcopy should make it a little faster, though I'm not sure we'd notice.